### PR TITLE
Courtney/feature/user messages

### PR DESCRIPTION
**Pull request by Courtney-Lippman**
# Description

A user message for the favorites page was created to inform the user that there are no games favorited yet.

The isLoading state, element, and functionality were added to the top 100 and details files. The styling for the loading messages was completed for both.  This will inform the user if the content for the application is still loading.

# Fixes # (issue)
#27 #32 

# Type of change

- [x]  Bug fix (non-breaking change which fixes an issue)
 
- [x]  New feature (non-breaking change which adds functionality)

- [x]  Breaking change (fix or feature that would cause existing functionality not to work as expected)

- [ ] This change requires a documentation update

-  [x] Styling

- [ ] Refactor

- [ ] Testing

# Where were the changes?
Details.css, Details.tsx, Top100.css, Top100.tsx, Favorites.tsx, Favorites.css

# Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [x]  I have commented my code, particularly in hard-to-understand areas (if applicable)
- [x]  I have made corresponding changes to the documentation (if applicable)
- [x]  My changes generate no new warnings

# Contributors (paired programming):
@Adelle-Pitsas @dgubko @Nagel29

### CODE REVIEW

**Review by Nagel29 — APPROVED** [review]
This loading animation looks awesome!

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -13,19 +13,23 @@ import { PageNotFound } from "../PageNotFound/PageNotFound";
...
+  const [isLoading, setisLoading] = useState<boolean>(true);
 
   useEffect(() => {
     Promise.resolve(getTop100())
       .then((data) => {
         setTop100(cleanTop100Data(data));
+        setisLoading(false);
       })
       .catch((response) => {
         console.log(response.status);
```
Comment (dgubko) [inline src/components/App/App.tsx:30]: we shall not forget to delete it)

File: src/components/Details/Details.tsx [hunk 2]
```diff
@@ -17,6 +18,7 @@ const Details = ({
   toggleFav: (id: string) => void;
 }) => {
   const [gameInfo, setGameInfo] = useState<CleanDetails>(Object);
+  const [isLoading, setisLoading] = useState<boolean>(true);
   const { id } = useParams();
 
   console.log("RIGHT HERE: ", top100[0]);
```
Comment (dgubko) [inline src/components/Details/Details.tsx:24]: this too

**Review by dgubko — APPROVED** [review]
Looks good, i love the loading screen while the page first loading

**Pull request merged**